lib/view/multi-tasks-view.js

```diff
@@ -58,7 +58,7 @@ class ListrReactiveTask {
                 subscriber.next(status);
             });
             this._eventEmitter.on('error', (error) => {
-                if (error && typeof error === 'string') {
```

Comment:
not sure what was the reason for this, but all errors that are not string would hang the deploy.